shared/language-specs/common.ts

```diff
@@ -0,0 +1,35 @@
+import { CommentStyle } from './spec'
```

Comment:
[minor nit] "common.ts" seems a little generic. What do you think about "comments.ts" to focus it on comment styles rather than suggesting a grab bag?

Reply:
Renamed.